src/components/chart/common-chart-options.js

```diff
@@ -30,9 +30,9 @@ class CommonChartOptions {
                     color: this.constants.labelColor, // Prevents the text from changing color on hover
                 },
                 itemStyle: {
-                    cursor: 'default', // ensures that it does not change to a hand (pointer) on hover.
+                    cursor: 'auto', // ensures that it does not change to a hand (pointer) on hover.
```

Comment:
we are already setting this in css

Reply:
I added it in the CSS as it didn't seem to work here - although it ought to. I can remove this line though.

Reply:
I've now removed this line.

Reply:
You have only commented it out

Reply:
Well that is embarrassing... now sorted.